schema/vulntology-json-schema-1.0-draft.json

```diff
@@ -149,7 +149,7 @@
                     "type": "object",
                     "properties": {
                         "hasImpactMethodType": {"const": "Context Escape"},
-                        "hasEscapeContext": {"$ref": "#/definitions/EscapeContext"}
+                        "hasEscapeContext": {"$ref": "#/definitions/Context"}
```

Comment:
This was a schema validation issue...unrelated to original ticket, but resolved while working in the schema. Introduced in #87